backend/medtagger/api/datasets/service_rest.py

```diff
@@ -0,0 +1,42 @@
+"""Module responsible for definition of Datasets service available via HTTP REST API."""
+from typing import Any
+
+from flask import request
+from flask_restplus import Resource
+
+from medtagger.api import api
+from medtagger.api.datasets import serializers, business
+from medtagger.api.security import login_required, role_required
+
+datasets_ns = api.namespace('datasets', 'Methods related with datasets')
+
+
+@datasets_ns.route('/')
```

Comment:
This is actually a mistake that we are doing in every service we have. It actually adds slash to the namespace making it incorrect in the first place. So instead of `/api/v1/scans/datasets` we have `/api/v1/scans/datasets/`.

After talking with @jpowie01 we also decided that this needs to be fixed in other places. Please correct it here and I will create separate story for fixing it in other services.